visual-recognition/src/main/java/com/ibm/watson/visual_recognition/v4/model/CreateCollectionOptions.java

```diff
@@ -25,10 +25,12 @@
   public static class Builder {
     private String name;
     private String description;
+    private TrainingStatus trainingStatus;
```

Comment:
The changes in this file need to be removed. TrainingStatus should not be a property here